tests/test_log.py

```diff
@@ -0,0 +1,38 @@
+import os
+import sys
+import unittest
+import yaml
+
+from mock import Mock, patch
+from patroni.config import Config
+from patroni.log import PatroniLogger
+
+
+class TestPatroniLogger(unittest.TestCase):
```

Comment:
what about the unit test for  backward compatibility with the previous way to configure logging ?

Reply:
It is basically here: https://github.com/zalando/patroni/pull/927/files#diff-423890b7e3c2aa4791f9e203b7f194cbR33